spyder/config/main.py

```diff
@@ -330,7 +330,6 @@
             ('completions',
              {
                'enable': True,
-               'kite_call_to_action': False,

```

Comment:
Since you're removing this option, you need to bump `CONF_VERSION` at the bottom of this file from `70.4.0` to `71.0.0`.